diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/spaces/IndentationRuleFixTest.kt

```diff
@@ -137,6 +140,76 @@ class IndentationRuleFixTest : FixTestBase("test/paragraph3/indentation",
             rulesConfigList = customConfig.asRulesConfigList())
     }
 
+    /**
+     * See [#1347](https://github.com/saveourtool/diktat/issues/1347).
+     */
+    @Test
+    @Tag(WarningNames.WRONG_INDENTATION)
+    fun `no whitespace should be injected into multi-line string literals (code matches settings, extendedIndent = true)`(@TempDir tempDir: Path) {
+        val defaultConfig = IndentationConfig("newlineAtEnd" to false)
+        val customConfig = defaultConfig.withCustomParameters(*extendedIndent(enabled = true))
+
+        lintMultipleMethods(
+            whitespaceInStringLiteralsContinuationIndent,
+            tempDir = tempDir,
+            rulesConfigList = customConfig.asRulesConfigList())
+    }
+
+    /**
+     * See [#1347](https://github.com/saveourtool/diktat/issues/1347).
+     */
+    @Test
+    @Tag(WarningNames.WRONG_INDENTATION)
+    fun `no whitespace should be injected into multi-line string literals (code matches settings, extendedIndent = false)`(@TempDir tempDir: Path) {
+        val defaultConfig = IndentationConfig("newlineAtEnd" to false)
+        val customConfig = defaultConfig.withCustomParameters(*extendedIndent(enabled = false))
+
+        lintMultipleMethods(
+            whitespaceInStringLiteralsSingleIndent,
+            tempDir = tempDir,
+            rulesConfigList = customConfig.asRulesConfigList())
+    }
+
+    /**
+     * See [#1347](https://github.com/saveourtool/diktat/issues/1347).
+     */
+    @Test
+    @Tag(WarningNames.WRONG_INDENTATION)
+    fun `no whitespace should be injected into multi-line string literals (mis-indented code reformatted, extendedIndent = true)`(@TempDir tempDir: Path) {
+        assumeTrue(testsCanBeMuted()) {
+            "Skipping a known-to-fail test"
```

Comment:
Shouldn't it be `@Disabled`? Or is there any advantage with this approach?

Reply:
Certain CI/CD servers (like _TeamCity_ &mdash; and I still have an urge to set up DiKTat on TeamCity) have a notion of a _muted failure_, i. e. a failing test can be marked as _muted_, so that the failure still gets reported but the build remains green as long as there're no new failures of other tests.

On the other hand, CI-wise, a `@Disabled` test is never run at all.

Additionally, I prefer to have two Run Configurations in IDEA &mdash; one with broken tests enabled and the other one with those disabled &mdash; w/o modifying the source code back and forth.

Anyway, this is a short-term measure &mdash; I don't plan to have a broken test in our codebase for a long time period.

Reply:
Thanks, TIL. Although, it seems like this feature makes build too tightly bound to a certain CI system? It seems for me that such feature should be implemented as a part of Junit engine.